src/server/cards/adventures/Disciple.js

```diff
@@ -0,0 +1,19 @@
+import Card from 'cards/Card';
+
+export default class Disciple extends Card {
+  static cost = new Card.Cost({ coin: 5 });
+  static types = new Set('Action', 'Traveller');
```

Comment:
All Travelers besides Page/Peasant should probably have
```js
static supplyCategory = 'nonSupply'
```
Or else the might get selected as one of the 10 kingdom cards.

Reply:
thats part of the supply now, still havent updated randomizer code though